extensions/amp-form/0.1/form-proxy.js

```diff
@@ -91,14 +91,15 @@ function createFormProxyConstr(win) {
   // Hierarchy:
   //   Node  <==  Element <== HTMLElement <== HTMLFormElement
   //   EventTarget  <==  HTMLFormElement
-  const prototypes = [
-    win.HTMLFormElement.prototype,
-    win.HTMLElement.prototype,
-    win.Element.prototype,
-    win.Node.prototype,
-    win.EventTarget.prototype,
+  const inheritance = [
+    win.HTMLFormElement,
+    win.HTMLElement,
+    win.Element,
+    win.Node,
+    win.EventTarget,
   ];
-  prototypes.forEach(function(prototype) {
+  inheritance.forEach(function(klass) {
+    const prototype = klass && klass.prototype;
```

Comment:
It's defined as an early break in ES5.1: https://www.ecma-international.org/ecma-262/5.1/#sec-12.6.4